airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/ImageCell.tsx

```diff
@@ -18,6 +18,10 @@ const Link = styled.a`
 `;
 
 const ImageCell: React.FC<ImageCellProps> = ({ imageName, link }) => {
+  if (!link || !link.length) {
```

Comment:
This is the cell in the tables on /settings/sources and /settings/destinations with the docker repository image name and the docs link.